translations/harbour-osmscout-server-es.ts

```diff
@@ -230,47 +230,47 @@
     <message>
         <location filename="../src/routingforhuman.cpp" line="98"/>
         <source>Drive along %along%</source>
-        <translation>Continua por la %along%</translation>
+        <translation>Conducir a lo largo%a lo largo%</translation>
```

Comment:
the spanish translate from carmenfdezb have a some error Translated, that why i decide get a pull request to u. i can fix now the string u talk about,